fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustGauge.tsx

```diff
@@ -30,6 +33,7 @@ interface ThrustGaugeProps {
 
 export class ThrustGauge extends DisplayComponent<ThrustGaugeProps> {
   private readonly sub = this.props.bus.getSubscriber<EwdSimvars>();
+  private readonly subArinc = this.props.bus.getSubscriber<Arinc429Values>();
```

Comment:
```suggestion
  private readonly sub = this.props.bus.getSubscriber<Arinc429Values & EwdSimvars>();
```
You can use multiple interfaces on the one subscriber no prob if you like (either way is fine though).

Reply:
Looks much nicer, thanks!